core/lib/config/src/configs/proof_data_handler.rs

```diff
@@ -2,18 +2,10 @@ use std::time::Duration;
 
 use serde::Deserialize;
 
-#[derive(Debug, Deserialize, Clone, Copy, PartialEq)]
-pub enum ProtocolVersionLoadingMode {
-    FromDb,
-    FromEnvVar,
-}
-
 #[derive(Debug, Deserialize, Clone, PartialEq)]
 pub struct ProofDataHandlerConfig {
     pub http_port: u16,
     pub proof_generation_timeout_in_secs: u16,
```

Comment:
BTW - what happens when someone still has PROOF_DATA_HANDLER_PROTOCOL_VERSION_LOADING_MODE set in their ENV variable ? it is silently ignored, or does the node crash ?

Reply:
ignored..

Reply:
Remove as many config as you can.